out_writeapi.go

```diff
@@ -330,6 +333,8 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 		log.Printf("Invalid Max_Queue_Requests parameter in configuration file: %s", err)
 		return output.FLB_ERROR
 	}
+	// Multiply floats, floor it, then convert it to integer for ease of use in Flush
+	queueRequestScaling := int(math.Floor(queueRequestScalingPercent * float64(queueSize)))
```

Comment:
For protection, should we also make sure this value never goes below, say, 10?

Also suggest to rename the variable queueRequestScaling ==> requestCountThreshold

Reply:
Done